docker/pwsh-remoting/Dockerfile

```diff
@@ -1,9 +1,3 @@
-FROM demisto/powershell-ubuntu:7.0.1.9103
+FROM devdemisto/powershell-ubuntu:7.0.3.11992
```

Comment:
we shouldnt use `devdemisto`

Reply:
Yes, it's only temporary until I'll push the new 7.0.3 version in this PR https://github.com/demisto/dockerfiles/pull/2809